[TOOLS-4352] NullPointException occurred during comment migration

Kang-dot opened the pull request:
http://jira.cubrid.org/browse/TOOLS-4352

- Purpose
fix NullPointException and add getColumnConfigIgnoreCase method

- Implementation
N/A

- Remarks
N/A

Kang-dot committed:
[TOOLS-4352] NullPointException occurred during comment migration

http://jira.cubrid.org/browse/TOOLS-4352

- add check null conditional statement
- add getColumnConfigIgnoreCase method for ignore target column name
case sensitive

hwany7seo commented on com.cubrid.cubridmigration.ui/src/com/cubrid/cubridmigration/ui/wizard/page/BaseConfirmationPage.java:
```diff
@@ -258,13 +258,21 @@ protected void setDDLText() {
-				if (tarCol.getComment() == null || sourceCol.getComment() != tarCol.getComment()){
-					tarCol.setComment(sourceCol.getComment());
+				SourceColumnConfig sourceCol = setc.getColumnConfig(tarCol.getName());
+				if (sourceCol.getComment() != null){
```
sourceCol  can be null.

hwany7seo commented on com.cubrid.cubridmigration.ui/src/com/cubrid/cubridmigration/ui/wizard/page/BaseConfirmationPage.java:
```diff
@@ -258,13 +258,21 @@ protected void setDDLText() {
 			Table tarTbl = cfg.getTargetTableSchema(setc.getTarget());
-			if (tarTbl.getComment() == null || tarTbl.getComment() != setc.getComment()) {
-				tarTbl.setComment(setc.getComment());
+			if (setc.getComment() != null) {
```
also setc can be null.

Kang-dot committed:
fix missing feature

- change getColumnConfig to getColumnConfigIgnoreCase
- change empty character to null

hwany7seo reviewed (commented):
please check comment.

airnet73 commented on com.cubrid.cubridmigration.ui/src/com/cubrid/cubridmigration/ui/wizard/page/BaseConfirmationPage.java:
```diff
@@ -258,13 +258,21 @@ protected void setDDLText() {
 				continue;
 			}
 			Table tarTbl = cfg.getTargetTableSchema(setc.getTarget());
```
If tarTbl is null, it seems necessary to check.

Kang-dot committed:
add Null check condition statement
